src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -349,62 +335,53 @@ In some cases adding content dynamically is an important part of user experience
   </figcaption>
 </figure>
 
-### Web fonts causing FOUT/FOIT
-
-Downloading and rendering web fonts can cause layout shifts in two ways:
-
-- The fallback font is swapped with a new font (FOUT - flash of unstyled text)
-- "Invisible" text is displayed until a new font is rendered (FOIT - flash of invisible text)
-
-The following tools can help you minimize this:
-
-- [`font-display`](https://developer.chrome.com/docs/lighthouse/performance/font-display/) allows you to modify the rendering behavior of custom fonts with values such as `auto`, `swap`, `block`, `fallback` and `optional`. Unfortunately, all of these values (except [`optional`](http://crrev.com/749080)) can cause a re-layout in one of the above ways.
-- The [Font Loading API](/optimize-webfont-loading/#the-font-loading-api) can reduce the time it takes to get necessary fonts.
+{% Aside important %}
+  If content is likely to take more than 500 milliseconds—for example it requires a network fetch—then reserving the expected space within that 500 millisecond timeframe and taking the impact of any future shift up front allows you to ensure any shifts will not be included in the CLS score.
+{% endAside %}
 
-More recommendations for reducing font-related CLS:
+### Animations
 
-- Using `<link rel=preload>` on the key web fonts: a preloaded font will have a higher chance to meet the first paint, in which case there's no layout shifting.
-- Combining `<link rel=preload>` and `font-display: optional`
+Changes to CSS property values can require the browser to react to these changes. A number of values trigger re-layout, paint and composite such as `box-shadow` and `box-sizing`. Try to avoid animating these.
 
-Read [Prevent layout shifting and flashes of invisible text (FOIT) by preloading optional fonts](/preload-optional-fonts/) for more details, and [Best practices for fonts](/font-best-practices/) for other font best practices.
+A number of CSS properties can be changed in a much more performant manner. For example, `transform` animations can be used to `translate`, `scale`, `rotate`, or `skew` without triggering a re-layout and so completely avoiding CLS.
 
-### Animations
+When animations are instead done by changing `top` and `left` CSS propertes instead of using `transform: translate`, layout shifts occur. This happens **even when the element being moved is in it's own layer and so does not cause shifts to other elements**. The net effect to the user may be the same, but one is counted as CLS, and one is not. This may seem like CLS should be changed to treat these the same, but there are other considerable performance benefits of using non-composited animations.
```

Comment:
Changed it to this:

> When animations are instead done by changing `top` and `left` CSS properties instead of using `translate`, layout shifts occur. This happens **even when the element being moved is in it's own layer and so does not cause shifts to other elements**. Composited animations via `translate` are exempt from CLS as they cannot impact other elements. There are also other considerable performance benefits of using non-composited animations since they do no cause re-layout and therefore are much less work for the browser.